invenio_requests/services/events/service.py

```diff
@@ -241,25 +243,86 @@ def search(
         """Search for events for a given request matching the querystring."""
         params = params or {}
         params.setdefault("sort", "oldest")
-
         expand = kwargs.pop("expand", False)
 
         # Permissions - guarded by the request's can_read.
         request = self._get_request(request_id)
         self.require_permission(identity, "read", request=request)
 
         # Prepare and execute the search
-        search = self._search(
+        search_result = self._search(
             "search",
             identity,
             params,
             search_preference,
             permission_action="unused",
             extra_filter=dsl.Q("term", request_id=str(request.id)),
             **kwargs,
+        ).execute()
+
+        return self.result_list(
+            self,
+            identity,
+            search_result,
+            params,
+            links_tpl=LinksTemplate(
+                self.config.links_search,
+                context={"request_id": str(request.id), "args": params},
+            ),
+            links_item_tpl=self.links_item_tpl,
+            expandable_fields=self.expandable_fields,
+            expand=expand,
         )
-        search_result = search.execute()
 
+    def focussed_list(
+        self,
+        identity,
+        request_id,
+        focus_event_id,
+        page_size,
+        expand=False,
```

Comment:
```suggestion
        expand=expand,
```
We should still allow the expand parameter as this controls the expansion of entities like `created_by`